sdk/python/feast/feature_service.py

```diff
@@ -0,0 +1,82 @@
+from datetime import datetime
+from typing import List, Optional, Union
+
+from feast.feature_reference import FeatureReference
+from feast.feature_table import FeatureTable
+from feast.feature_view import FeatureView
+from feast.protos.feast.core.FeatureService_pb2 import (
+    FeatureService as FeatureServiceProto,
+)
+from feast.protos.feast.core.FeatureService_pb2 import (
+    FeatureServiceMeta,
+    FeatureServiceSpec,
+)
+
+
+class FeatureService:
+    name: str
+    features: List[FeatureReference]
+    created_timestamp: Optional[datetime] = None
+    last_updated_timestamp: Optional[datetime] = None
+
+    def __init__(
+        self,
+        name: str,
+        features: List[Union[FeatureTable, FeatureView, FeatureReference]],
```

Comment:
In which cases would users provide references and in which cases tables/views?

Reply:
Per the RFC, we want to support both of these cases:

```
from feast.feature_service import FeatureService, RawFeature
from my_repo.feature_views import shop_raw, customer_sales

feature_service = FeatureService(
    name="my_super_model_v1",
    features=[
        shop_raw,
        customer_sales[['average_order_value', 'max_order_value']],
    ],
)
```

Reply:
Isn't it more complicated to support different types though? What if we just had references and not views/tables?

Reply:
They are supported only in the `__init__` method - we convert everything  to a reference, and that's what is stored in the FeatreService proto. I think that provides a good level of abstractions for users.